common/bsonutil/bsonutil.go

```diff
@@ -477,14 +477,58 @@ func MtoD(m bson.M) bson.D {
 // but would return an error if it cannot be reversed by bson.UnmarshalExtJSON.
 //
 // It is preferred to be used in mongodump to avoid generating un-reversible ext JSON.
-func MarshalExtJSONReversible(val interface{}, canonical bool, escapeHTML bool) ([]byte, error) {
+func MarshalExtJSONReversible(
```

Comment:
I found it a bit confusing that this func was changing in this PR, but I eventually realized this was purely cosmetic.

In these sorts of cases, I think there's a few ways to handle this:

1. Make a separate PR for cosmetic changes.
2. Keep it in the same PR, but call it out in the PR description so reviewers know why this is here. You can also add your own comment to the PR calling this out, which is helpful for reviewers.